### ISSUE TITLE
[menu] Mobile Services>Device Security>Setting up is 404

### ISSUE THREAD

**Issue by pmacko1** [aerogear/mobile-docs#456]
## Description
[menu] Mobile Services>Device Security>Setting up is 404

## Link to Page/Section where Issue was seen
nav.adoc

## Suggested edit
point it to `Configuring your Development Environment for the Device Security Service`. Blocked by another issue -> xref to that paragraph is `#con_introduction-to-device-security_additional-resources-con_introduction-to-device-security`-> this does not look ok

**Comment by pmacko1** [comment]
adding an anchor solved this
